proposals/lip-0061.md

```diff
@@ -85,10 +85,10 @@ In general, for a height `h`, the certificate threshold could be chosen within t
 floor(1/3*aggregateBFTWeight(h))+1 <= certificateThreshold(h) <= aggregateBFTWeight(h),
 ```
 
-where `aggregateBFTWeight(h)` is the sum of BFT weights at height `h` and `certificateThreshold(h)` is the certificate threshold at height `h`. This allows for a sidechain to choose different trade-offs between satisfying certification liveness (generating certificates with enough signatures) and state transition validity (certificates attest a valid sidechain history):
+where `aggregateBFTWeight(h)` is the sum of BFT weights at height `h` and `certificateThreshold(h)` is the certificate threshold at height `h`. This allows for a sidechain to choose different trade-offs between satisfying cross-chain certification liveness (generating certificates with enough signatures) and state transition validity (certificates attest a valid sidechain history):
```

Comment:
I would say that `cross-chain` does not always need to be added if the context is clear. For instance, I think it is fine to just say `message` if it is clear that it is a cross-chain message. This helps with readability without losing any precision as the context is clear.

Similarly, I would keep sidechain validators as "validators of the sidechain" is just a less readable synonym.

Reply:
I reverted the changes related to "cross-chain".